### PR TITLE
Podcast Page Header Refresh

### PR DESCRIPTION
**Pull request by jsunsawyer**
@jerodsanto I've added a various `TODO` comments for various things that you'll want to address before making this live.

The avatar list has alternate styles when there are 3 or fewer people. For now, we're only applying this class (and subsequently these styles) on the podcast page itself, not on individual episodes.

Also, we're testing out left aligned content at mobile. Check it out and let me know if you want anything adjusted.

### CODE REVIEW

**Comment by jerodsanto** [comment]
@jsunsawyer I just pushed up some commits that take care of most of the TODOs. A few thoughts and things to figure out:

1. We can't merely split the names in two because some people have 'abnormal' names, such as `Jaana B. Dogan` and `Mireille B. Reece, Psy.D`. Is there a way we can accomplish this stacked effect with html/css trickery instead of server-side code?
2. This header is so significant that it doesn't make sense on anything but page 1 of a podcast index. It's bad for recommended, popular, pagination, etc. Instead of making it work somehow, I'm curious if we could do a separate 'minimal' page header for these sub-pages for a podcast. Would love your thoughts on this @codyjames 
3. Left aligning on mobile: I'm not in love. Maybe we could center align 'assets' and left align just the copy?

File: lib/changelog_web/templates/podcast/_subscribe_buttons.html.eex [hunk 1]
```diff
@@ -4,6 +4,10 @@
   <%= if @podcast.apple_url do %>
     <%= link("Apple", to: @podcast.apple_url, class: "show-header-subscribe-buttons-item show-header-subscribe-buttons-item--apple", title: "Subscribe on Apple") %>
     <%= link("Overcast", to: subscribe_on_overcast_url(@podcast), class: "show-header-subscribe-buttons-item show-header-subscribe-buttons-item--overcast", title: "Subscribe on Overcast") %>
+  <% end %>
+  <%= if @podcast.spotify_url do %>
+    <%# TODO add Spotify image asset and enable this %>
```
Comment (jerodsanto) [inline lib/changelog_web/templates/podcast/_subscribe_buttons.html.eex]: @jsunsawyer please add this to your todo list before we ship this

**Comment by codyjames** [comment]
Perhaps we just show the First Name and have the full name live in the anchor title?

**Comment by codyjames** [comment]
We can surely work up a minimal version for those paged and tabbed views.

**Comment by jerodsanto** [comment]
> Perhaps we just show the First Name and have the full name live in the anchor title?

I don't understand the visual implications of that. Will it just say 'Jerod' and 'Adam' next to our avatars when you visit /podcast?

> We can surely work up a minimal version for those paged and tabbed views.

ok let's do that I think it's a good idea all around. Better than our current #anchor links even.

**Comment by codyjames** [comment]
For podcasts with many guests it would just say their first name. On hover it'd just be the default browser title tag displaying their full name. For podcasts with few guests, we'd still show the full name.

**Comment by jerodsanto** [comment]
What would be the cut off? (Also I assume you mean hosts...) We have tooltips don't we? Could we use those in addition to anchor titles?

**Comment by jsunsawyer** [comment]
@jerodsanto Pushed up a few updates that should address your issues. Let me know if you need any adjustments.

**Comment by jerodsanto** [comment]
🚀 🤘

**Pull request merged**